src/chrome/content/rules/Rpgmakerweb.com.xml

```diff
@@ -22,7 +22,7 @@
 	<target host="rpgmakerweb.com" />
 
 
-	<securecookie host=".+" name="^optimizely" />
+	<securecookie host=".+" name=".+"/>
 
```

Comment:
this change is strange...

Reply:
Apparently this line is the culprit 

https://github.com/EFForg/https-everywhere/blob/a2bcade6b8c4bca45ac8cbb93021077121da0dad/utils/trivialize-rules/trivialize-cookie-rules.js#L68